docs/core/index.md

```diff
@@ -68,3 +68,9 @@ Log out of the currently authenticated partner organization and store. The `logo
 $ shopify logout
 ```
 
+## `config analytics`
+
+Configure anonymous usage reporting by enabling or disabling the analytics flag
```

Comment:
Minor, but I'd maybe say "enabling or disabling the analytics". The flag is the is what we use to configure it it.

Reply:
😅 good point!